package-lock.json

```diff
@@ -1,2180 +1,7 @@
 {
   "name": "akkaserverless-javascript-sdk",
-  "lockfileVersion": 2,
```

Comment:
We have changed this everywhere else to `"lockfileVersion": 1`, why are we removing it here?